src/main/resources/templates/searchUpDownResults.html

```diff
@@ -4,9 +4,48 @@
 <head>
   <title>Search Results</title>
   <div th:replace="fragments/bootstrap_head.html"></div>
+  <script>
+  function onLoad() {
+    $(document).ready( function() {
```

Comment:
By the way, this is definitely _not_ the most modern/optimal way to handle this. But with limited time, it's the easiest way to get it to work, and should work on even older browsers.

Reply:
this is rlly good nice